doc/basics.md

```diff
@@ -329,7 +344,7 @@ void recursive_print_json(T&& element) {
         cout << ",";
       }
       // key() returns the unescaped key, if we
```

Comment:
Maybe rephrase it to `key() returns the original key` or `key() returns the key as specified in the document`?

Otherwise the sentence is weird: `key() returns the unescaped key, if we want the unescaped key, ..`

Reply:
I have improved the wording. It was terrible, thanks.